tools/reviewer/job/tasks/ReviewerMetadataUpdaterTask.java

```diff
@@ -85,6 +96,11 @@ private void uploadReviewerRegistryToFirestore(ReviewerRegistry registry) {
         REVIEWER_COLLECTION, REVIEWER_REGISTRY_DOCUMENT_NAME_BIN, registry);
   }
 
+  private void uploadReviewerConfigToFirestore(ReviewerConfig config) {
+    firestoreClient.setProtoDocument(
```

Comment:
I don't think it's worth creating a method for just this command.

Reply:
Why not?
I think it keeps the code DRY and easy to understand.
Also, if we don't need `uploadReviewerConfigToFirestore()`, why did my predecessors create `uploadReviewerRegistryToFirestore()`?